aiounifi/interfaces/vouchers.py

```diff
@@ -0,0 +1,43 @@
+"""Hotspot vouchers as part of a UniFi network."""
+
+from ..models.api import TypedApiResponse
+from ..models.voucher import (
+    Voucher,
+    VoucherCreateRequest,
+    VoucherDeleteRequest,
+    VoucherListRequest,
+)
+from .api_handlers import APIHandler
+
+
+class Vouchers(APIHandler[Voucher]):
+    """Represents Hotspot vouchers."""
+
+    obj_id_key = "_id"
+    item_cls = Voucher
+    api_request = VoucherListRequest.create()
+
+    async def create(self, voucher: Voucher) -> TypedApiResponse:
+        """Create voucher on controller."""
+        return await self.controller.request(
+            VoucherCreateRequest.create(
+                number=1,
```

Comment:
Number is a required item to create an instance of Voucher, is there a reason 1 is hardcoded?

Reply:
Number is not an attribute of Voucher. Number is the number of vouchers to be create in one request

Reply:
Alright, I would just add the default value  to the create request and remove this instead.
```suggestion
```